src/components/Form/SelectFIeld.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './formFields.css';
+
+interface SelectOption {
+  value: string;
+}
+
+type SelectProps = {
+  label: string;
+  value: string;
```

Comment:
Also here is string for value type? Try to make it generic. Read about generics in Typescript.

It goes something liek SelectProps<T> = {value: T}